### ISSUE TITLE
CocHintHighlight syntax highlight group doesn't work anymore

### ISSUE THREAD

**Issue by antaed** [neoclide/coc.nvim#3139]
## versions

vim version: NVIM v0.4.3
node version: v16.2.0
coc.nvim version: 0.0.80-69e029ed9e
coc.nvim directory: /root/.local/share/nvim/plugged/coc.nvim
term: xterm-256color
platform: linux

## Log of coc.nvim

2021-06-11T15:32:32.833 INFO (pid:1611) [services] - registered service "vimlsp"
2021-06-11T15:32:32.834 INFO (pid:1611) [services] - vim language server state change: stopped => starting
2021-06-11T15:32:32.836 INFO (pid:1611) [plugin] - coc.nvim 0.0.80-69e029ed9e initialized with node: v16.2.0 after 123ms
2021-06-11T15:32:33.071 INFO (pid:1611) [language-client-index] - vimlsp started with 1622
2021-06-11T15:32:33.198 INFO (pid:1611) [services] - vim language server state change: starting => running
2021-06-11T15:32:33.201 INFO (pid:1611) [services] - service vimlsp started
2021-06-11T15:36:03.015 INFO (pid:1611) [attach] - receive notification: showInfo []

## Describe the bug

After recently updating Neovim and all plugins, including coc.nvim, I noticed the **CocHintHighlight** syntax highlight group now has a different color than set in my theme. I even tried setting it manually but it still didn't work, as if the group doesn't exist. I also tried the command on a different highlight group and it worked.

To summarize:
`:hi CocErrorHighlight guifg=#00ff00` - works
`:hi CocHintHighlight guifg=#00ff00` - does not work

## Reproduce the bug

I reproduced the bug on a different machine by updating Neovim, and all plugins, including coc.nvim.

**Comment by chemzqm** [comment]
You need diagnostic with type set to `hint` to see it.

**Comment by antaed** [comment]
> 
> 
> You need diagnostic with type set to `hint` to see it.

Sorry to bother you but may I ask how can I do that? Thanks

**Comment by chemzqm** [comment]
Try language server that provide hint diagnostics

**Comment by antaed** [comment]
> 
> 
> Try language server that provide hint diagnostics

coc-phpls seems to provide hint diagnostic as seen in the image below. 
Prior to some recent updates, the highlight color was the same as seen in the signcolumn (instead of brown).
I should probably address the issue to coc-phpls as well.

![image](https://user-images.githubusercontent.com/30736986/122121710-9ea27400-ce34-11eb-802c-53ab6ee918eb.png)
